Dockerfile

```diff
@@ -1,5 +1,5 @@
 ############# builder
-FROM golang:1.15.8 AS builder
+FROM golang:1.16.2 AS builder
```

Comment:
Do you also want to update the alpine image?

Reply:
done

Reply:
Actually should I change it to `eu.gcr.io/gardener-project/3rd/alpine` ?

Reply:
IIUC, this is no longer needed now that we use Kaniko